Sources/Build/llbuild.swift

```diff
@@ -151,47 +172,59 @@ public struct LLBuildManifestGenerator {
             }
         }
 
+        var buildTarget = Target(name: target.target.targetName)
```

Comment:
Don't think os. The `targetName` always contains a suffix `.module`/`.dylib`/`.test`/`.a`/`.exe`.